stories/molecules/Pill.stories.tsx

```diff
@@ -61,6 +67,91 @@ export const _PillButton: PillButtonStory = {
     ),
 };
 
+type PillButtonEnhancedStory = StoryObj<typeof PillButtonEnhanced>;
+
+export const _PillButtonEnhanced: PillButtonEnhancedStory = {
+    name: 'PillButtonEnhanced',
+    args: {
+        name: 'Pill name',
+        content: 'value',
```

Comment:
It's strange, but why the content is JSON string:
![Screenshot 2024-11-08 at 11 42 08](https://github.com/user-attachments/assets/0fff9e19-e2e3-4283-8573-9815f063f040)

Reply:
I think because this prop accepts any ReactNode. This is the definition taken from the current PillButton.